src/static/App.css

```diff
@@ -1,24 +1,23 @@
-.App {
-  text-align: center;
+.contentContainer {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
 }
 
-.App-logo {
-  animation: App-logo-spin infinite 20s linear;
-  height: 80px;
+.signupButton {
+  margin-top: 16px;
 }
 
-.App-header {
-  background-color: #222;
-  height: 150px;
-  padding: 20px;
-  color: white;
+#signupText {
+  margin: 0;
+  margin-top: 12px;
 }
 
-.App-intro {
-  font-size: large;
-}
-
-@keyframes App-logo-spin {
-  from { transform: rotate(0deg); }
-  to { transform: rotate(360deg); }
+#signupForm {
```

Comment:
nit: typically its better to use a class selector over an id selector. There are a few reasons as far as how this affects cascading styles 

https://css-tricks.com/a-line-in-the-sand/